auto_forwarder.py

```
@@ -363,6 +502,94 @@ def _process_album(self, grouped_id):
+        """Retrieves the maximum of the Telegram-tracked read ID and plugin's internal last_seen_inbox_id."""
+        try:
+            # Get Telegram's read position
+            dialog = get_messages_controller().getDialog(chat_id)
+            telegram_read_id = getattr(dialog, 'read_inbox_max_id', 0) if dialog else 0
+            
+            # Get plugin's internal tracking
+            internal_tracking_key = f"last_seen_{chat_id}"
+            internal_read_id = int(self.get_setting(internal_tracking_key, "0"))
+            
+            return max(telegram_read_id, internal_read_id)
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)